guides/micronaut-docker-image/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Building a Docker Image of your Micronaut application",
+  "intro": "Micronaut Build plugins offer several ways to build Docker Images - JAR, Native Executable, CRaC.",
+  "authors": ["Sergio del Amo"],
+  "categories": ["Distribution"],
+  "publicationDate": "2023-02-12",
+  "apps": [
+    {
+      "name": "default",
+      "features": ["graalvm","crac","micronaut-aot"]
```

Comment:
I don't see the graalvm feature:

<img width="2129" alt="image" src="https://user-images.githubusercontent.com/153880/218453376-43a2c3fd-bce0-4eec-a442-ac6ef9eba323.png">

Perhaps the PDF is not up-to-date?